index.md

```diff
@@ -20,7 +20,6 @@ ms.assetid:
     <section id="hero-content" class="graph">
         <h1>.NET Documentation</h1>
         <h2>Welcome to .NET! This is the home of the technical documentation for .NET, C#, F# and Visual Basic, including basic concepts, getting started instructions, tutorials and samples. Start reading and exploring, to learn .NET for the first time or to find an answer to today's challenge.</h2>
-        <h3>New to .NET? Visit the <a href="">.NET Products page</a> &rarr;</h3>
```

Comment:
Ah, I meant it to go to https://dot.net. Not sure why I didn't add that.

That said, I don't feel strongly about keeping it.